packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -171,19 +164,12 @@ export const messageThreadSelector: MessageThreadSelector = createSelector(
         )
         .filter((message) => message.content && message.content.message !== '') // TODO: deal with deleted message and remove
         .map((message) => {
-          const readBy: { id: string; name: string }[] = getParticipantsWhoHaveReadMessage(
-            message,
-            readReceiptForEachSender
-          );
-
           return memoizedFn(
             message.id ?? message.clientMessageId,
             message,
             userId,
             message.createdOn <= latestReadTime,
-            isLargeGroup,
-            readBy.length,
```

Comment:
We will still need to populate these fields even if we do not use them to prevent any breaking change (will be breaking if a customer relies on this prop but it is suddenly undefined)

Reply:
> We will still need to populate these fields even if we do not use them to prevent any breaking change (will be breaking if a customer relies on this prop but it is suddenly always undefined)
readBy.length and readBy are two fields I added when building read by information, just removing them now so it will not introduce new perf regression

Reply:
I am getting confused on an issue where it looks like we got sign off on something incorrect: https://github.com/Azure/communication-ui-library/pull/1691#discussion_r837937832

Hold off on this comment for a bit 😅

Reply:
This is now resolved.